src/T2DMap.cpp

```diff
@@ -4611,13 +4611,42 @@ void T2DMap::wheelEvent(QWheelEvent* e)
     if (e->modifiers() & Qt::ControlModifier) { // Increase rate 10-fold if control key down - it makes scrolling through
                                                 // a large number of items in a listwidget's contents easier AND this make it
                                                 // easier to zoom in and out on LARGE area maps
-        delta *= 10;
+        delta *= 5;
     }
     if (delta != 0) {
         mPick = false;
-        int oldZoom = xyzoom;
-        xyzoom = qMax(3, xyzoom + delta);
+        qreal oldZoom = xyzoom;
+        xyzoom = qMax(3.0, xyzoom * pow(1.07, delta));
+
         if (oldZoom != xyzoom) {
+            const float widgetWidth = width();
+            const float widgetHeight = height();
+            float xs = 1.0;
```

Comment:
Don't you need a `f` suffix to the `1.0` to register it as a literal `float` constant rather than a `double` (the default) one.

Reply:
It's a constant with no loss of precision when converted to a `float`, so the compiler does the right thing. No conversion warning.